src/components/EnterpriseApp/index.jsx

```diff
@@ -82,6 +82,7 @@ class EnterpriseApp extends React.Component {
       enableAnalyticsScreen,
       enableReportingConfigurationsScreen,
       enablePortalLearnerCreditManagementScreen,
+      enterpriseGroupsV2,
```

Comment:
this field `enterpriseGroupsV2` is nested in the `enterpriseFeatures` object on line 88; otherwise the value will always return `false here`.